worker/src/RTC/DtlsTransport.cpp

```diff
@@ -913,7 +1011,7 @@ namespace RTC
 			this->timer->Stop();
 
 			// Process the handshake just once (ignore if DTLS renegotiation).
-			if (!wasHandshakeDone && this->remoteFingerprint.algorithm != FingerprintAlgorithm::NONE)
```

Comment:
Is this condition wrapped into some `if (this->remoteFingerprint.algorithm.has_value())`? It's hard to see in the diff.

Reply:
Nop, `FingerPrint::algorithm` is not optional. If the handshake was done it must have a valid value.

Reply:
ok